features/support/env.rb

```diff
@@ -52,6 +53,16 @@ def self.connection
 end
 ActiveRecord::Base.shared_connection = ActiveRecord::Base.connection
 
+module MutexLockedQuerying
```

Comment:
Can you add a comment about what this is for?

Reply:
The commit message offers an explanation: https://github.com/concord-consortium/rigse/pull/549/commits/700eb071602538e29671118232fc3749859d732b

> Because of the shared active record db connection, we saw errors like the following upon migrating from firefox to chrome
> 
> Mysql2::Error: This connection is in use by: #<Thread:0x0000000d5bada8@/usr/local/lib/ruby/2.2.0/webrick/server.rb:283 sleep>
> 
> Adding this mutex resolves the issue. Once the app has been upgraded to Rails 5.1, all of this can be removed since database connections are shared there by default (yay!)
>